cmd/utils/flags.go

```diff
@@ -383,6 +383,10 @@ var (
 		Name:  "cache.noprefetch",
 		Usage: "Disable heuristic state prefetch during block import (less CPU and disk IO, more time waiting for data)",
 	}
+	CacheNoPreimageFlag = cli.BoolFlag{
+		Name:  "cache.nopreimage",
+		Usage: "Disable recording the SHA3/keccak preimages of trie keys",
```

Comment:
`--cache.preimage` with a default of `true`? That way we don't need to change the API on the next release, only flip the default value.

Reply:
Let's also plural :) `preimageS`

Reply:
The BoolFlag seems doesn't have the default value. So the default is False (by default)

Reply:
Ah, we have another Flag type called `BoolTFlag`